spec/hosts/roles/hexrot_spec.rb

```diff
@@ -32,16 +32,85 @@
           it { is_expected.to contain_class('docker') }
 
           %w[
+            anaconda
```

Comment:
This duplicates line below including ` it { is_expected.to contain_class('anaconda').with_anaconda_version('Anaconda3-2023.07-2') }`

Reply:
changed